backend/src/scripts/utility/README.md

```diff
@@ -0,0 +1,50 @@
+# Utility Suite
+The utility script consist of a library of helpful commands to extract or update the data within the application.
+
+To see the list of commands and how to run them, run `yarn run-script --help` on the [backend root folder](../../../)
+
+## Scheduler
+[The scheduler](./scheduler.js) is an executable script. Depending on the [configuration](scheduler.config.js), it will execute certain script(s) within the utility suite. It is designed to be hooked onto a job scheduler (e.g. `cron`) to be executed on a daily basis.

```

Comment:
The configuration link doesn't seem to work

Reply:
Fixed